### PR TITLE
Small changes to names of the tutorials

### PR DESCRIPTION
**Pull request by MartaDubno**

### CODE REVIEW

File: src/get-started/data-proofs.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Service with Data Proofs
+# Advanced Cryptocurrency Tutorial: Service with Data Proofs
 
 This tutorial is an extended version of the
-[service creation tutorial](create-service.md).
+[cryptocurrency tutorial](create-service.md).
```
Comment (Maria-Nosyk) [inline src/get-started/data-proofs.md]: Cryptocurrency Tutorial

File: src/get-started/light-client.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Interacting with Exonum Services
+# Exonum Light Client Tutorial
 
 In this tutorial we describe how to use the light client to interact with Exonum
 services. The tutorial extends other tutorials:
-[How to Create a Service with Data Proofs](data-proofs.md)
-and, partly, [Timestamping Demo][timestamping-demo].
+[cryptocurrency advanced](data-proofs.md)
+and, partly, [timestamping][timestamping-demo].
```
Comment (Maria-Nosyk) [inline src/get-started/light-client.md]: Cryptocurrency Advanced
Timestamping

File: src/get-started/light-client.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ const TransferFunds = Exonum.newMessage({
 ```
 
 As you might know from our very first tutorial
-[How to Create a Service](create-service.md),
+[cryptocurrency tutorial](create-service.md),
```
Comment (Maria-Nosyk) [inline src/get-started/light-client.md]: Cryptocurrency Tutorial

File: mkdocs.yml [hunk 4]
```diff
@@ -51,13 +51,13 @@ pages:
     # Installation guide
     - 'Install': 'get-started/install.md'
     # Writing one's first Exonum service
-    - 'Create Service': 'get-started/create-service.md'
+    - 'Cryptocurrency Service': 'get-started/create-service.md'
```
Comment (slowli) [inline mkdocs.yml]: Maybe, it would make sense to rename this and the following pages to "Cryptocurrency Tutorial" / "Advanced Cryptocurrency Tutorial" ?

**Pull request merged**